plugins/node/opentelemetry-instrumentation-aws-lambda/src/instrumentation.ts

```diff
@@ -231,6 +230,28 @@ export class AwsLambdaInstrumentation extends InstrumentationBase {
     this._tracerProvider = tracerProvider;
   }
 
+  private _getActiveSpanProcessor() {
```

Comment:
I wonder if you want to cache the unproxied tracer provider?

Reply:
I refactored and hopefully it is something we both agree is better. "Caches" without having to deal with multiple field states as we don't need a `_traceProvider` field, just `_forceFlush`. Since you helped me know what the problem is with `this`, I was able to just use `bind` to help with keeping nastiness in one method.